.github/workflows/release-linux.yml

```diff
@@ -23,30 +23,60 @@ jobs:
       run: |
         tar -xf source.tar.gz --strip-components=1
 
+    - name: Enable vcpkg cache
+      uses: actions/cache@v3
+      with:
+        path: /vcpkg/installed
+        key: ubuntu-20.04-vcpkg-release-0 # Increase the number whenever dependencies are modified
+        restore-keys: |
+          ubuntu-20.04-vcpkg-release
+
     - name: Install dependencies
       run: |
-        echo "::group::Install dependencies"
+        echo "::group::Install system dependencies"
+        # ICU is used as vcpkg fails to install ICU. Other dependencies
+        # are needed either for vcpkg or for the packages installed with
+        # vcpkg.
         yum install -y \
-          fontconfig-devel \
-          freetype-devel \
-          libcurl-devel \
           libicu-devel \
-          libpng-devel \
-          lzo-devel \
-          SDL2-devel \
+          perl-IPC-Cmd \
           wget \
-          xz-devel \
-          zlib-devel \
+          zip \
           # EOF
         echo "::endgroup::"
 
+        # We use vcpkg for our dependencies, to get more up-to-date version.
+        echo "::group::Install vcpkg and dependencies"
+        git clone --depth=1 https://github.com/microsoft/vcpkg
+        (
+          cd vcpkg/
+          ./bootstrap-vcpkg.sh -disableMetrics
+
+          # Make Python3 available for other packages.
+          ./vcpkg install python3
+          ln -sf $(pwd)/installed/x64-linux/tools/python3/python3.10 /usr/bin/python3
```

Comment:
Won't this be an issue if vcpkg installs another version ?

Reply:
Good catch! Fixed :)